integrations/extensions/docs/elasticsearch-install-and-setup/text_embedding_deploy_and_use.md

```diff
@@ -219,4 +223,11 @@ curl -X GET "${ES_URL}/${ES_EMBEDDING_INDEX_NAME}/_search" \
     "text"
   ]
 }'
-```
\ No newline at end of file
+```
+
+## What's next?
+Now that you've successfully deployed your text embedding model in Elasticsearch, here are the recommended next steps:
+### 1. Configure your assistant for document search
+Use the vector representations generated by your deployed model to enhance document search in your assistant. Follow the instructions provided [here](../../starter-kits/elasticsearch/README.md) to integrate Elasticsearch as an extension into your assistant.
+### 2. Use search results as input context to an LLM
+Follow the example provided [here](../../starter-kits/language-model-conversational-search/README.md) to leverage the search results from Elasticsearch as input context to a watsonx large language model (LLM) to generate contextually relevant responses to user queries.
```

Comment:
Thanks, Reza, for making the updates. I would add another option here for supporting the built-in search integration, which is about using the right query in the configuration.

Reply:
Thanks, Zach! I've added that option as well. Note that there is a link to a section in the README file put together as part of PR https://github.com/watson-developer-cloud/assistant-toolkit/pull/253 which will work once that PR is merged.